addons/l10n_do/i18n/es.po

```diff
@@ -0,0 +1,3473 @@
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_11100200
+#: model:account.account.template,name:l10n_do.l10n_do_11100200
+msgid "Prepaid rents"
+msgstr "Alquileres pagados por Anticipado"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_21030502
+#: model:account.account.template,name:l10n_do.l10n_do_21030502
+msgid "Advances ISR payable"
+msgstr "Anticipos ISR por Pagar"
+
+#. module: l10n_do
+#: model:account.group,name:l10n_do.2_account_group_2203
+#: model:account.group.template,name:l10n_do.account_group_2203
+msgid "Customer Advances and Unidentified Payments"
+msgstr "Anticipos de Clientes y Pagos no Identificados"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_22030100
+#: model:account.account.template,name:l10n_do.l10n_do_22030100
+msgid "Customer Advances or Guarantees"
+msgstr "Anticipos o Garantías de Clientes"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_11100600
+#: model:account.account.template,name:l10n_do.l10n_do_11100600
+msgid "Advances for 1% Assets"
+msgstr "Anticipos para 1% Activos"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_11100500
+#: model:account.account.template,name:l10n_do.l10n_do_11100500
+msgid "Advances for Expenses"
+msgstr "Anticipos para Gastos"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_11100400
+#: model:account.account.template,name:l10n_do.l10n_do_11100400
+msgid "Advances for income tax"
+msgstr "Anticipos para ISR"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_11100300
+#: model:account.account.template,name:l10n_do.l10n_do_11100300
+msgid "Advances for Insurance"
+msgstr "Anticipos para Seguros"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_21020800
+#: model:account.account.template,name:l10n_do.l10n_do_21020800
+msgid "Pension Fund (AFP) Contribution Payable"
+msgstr "Aporte a Fondo de Pensiones (AFP) por Pagar"
+
+#. module: l10n_do
+#: model:account.group,name:l10n_do.2_account_group_610101
+#: model:account.group.template,name:l10n_do.account_group_610101
+msgid "Social Security Contributions"
+msgstr "Aportes a la Seguridad Social"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_61030107
+#: model:account.account.template,name:l10n_do.l10n_do_61030107
+msgid "Counseling (Individual)"
+msgstr "Asesorías (P. Física)"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_61030207
+#: model:account.account.template,name:l10n_do.l10n_do_61030207
+msgid "Counseling (Legal)"
+msgstr "Asesorías (P. Jurídica)"
+
+#. module: l10n_do
+#: model:account.account,name:l10n_do.2_l10n_do_11100100
+#: model:account.account.template,name:l10n_do.l10n_do_11100100
+msgid "Advance to Suppliers"
+msgstr "Avance a Proveedores"
+
+#. module: l10n_do
+#: model:account.report.column,name:l10n_do.tax_report_balance
+msgid "Balance"
+msgstr "Balance"
```

Comment:
Maybe find a proper translation for this, when using deepl ==> "Saldo", "Balanza" ?